README.md

```diff
@@ -257,6 +259,16 @@ Notes:
 
 ### Special classes within teacher content
 
+####Teacher Content with NGSS tag
+
+```html
+
+<note class="os-teacher">
+  <label>Teacher Edition</label>
+  <p>NGSS HS-PS2-1:<span class="ost-tag-ngss-k12phys-hs-ps2-1 ost-ngss-def">Students who demonstrate understanding can: Analyze
```

Comment:
@philschatz we were trying to structure it more like how the TEKS are defined; the "NGSS HS-PS2-1" seems easier for the XML teams to locate than a phrase with a colon after it.  Technically W&N's markup is inconsistent between TEKS defined and NGSS defined...I could see us going either way.  Thoughts? (@kjdav ?)